lib/views/commit-view.js

```diff
@@ -78,7 +83,7 @@ export default class CommitView extends React.Component {
         return;
       }
 
-      const fakeEvent = new FakeKeyDownEvent(keyCode);
+      const fakeEvent = getFakeKeyDownEvent(keyCode);
```

Comment:
Sweet, thanks for fixing the broken snapshotting @annthurium! ✨ 

One minor suggestion with efficiency in mind - with this approach we are creating a new class `FakeKeyDownEvent` every time `proxyKeyCode` gets called. 

We can do this just once if we do something like this:

have a global variable:
```
let FakeKeyDownEvent;
```

and in `proxyKeyCode` check to see if it exists, creating it if it doesn't:
```
if (!FakeKeyDownEvent) {
  FakeKeyDownEvent = class FakeKeyDownEvent extends CustomEvent {
    constructor(kCode) {
      super('keydown');
      this.keyCode = kCode;
    }
  }
}

const fakeEvent = new FakeKeyDownEvent(keyCode);
```

Reply:
good suggestion!  Will fix.